### PR TITLE
UpgradePlanModal

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What / Why

Integrated UpgradePlanModal with new DS to replace the old one
+ tests + storybook

## Demo

Capture d’écran 2021-09-03 à 16 24 08

### CODE REVIEW

File: packages/core/admin/admin/src/components/UpgradePlanModal /UpgradePlanModal.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!--- UpgradePlanModal.stories.mdx --->
```
Comment (bulby97) [inline packages/core/admin/admin/src/components/UpgradePlanModal /UpgradePlanModal.stories.mdx:1]: Maybe I'm missing something, do we have storybook in the core admin ?. I thought it was installed in parts and helper-plugin.

File: packages/core/admin/admin/src/components/UpgradePlanModal /index.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+  img:not(:first-of-type) {
+    width: ${130 / 16}rem;
+    margin-left: 12%;
+    margin-right: ${20 / 16}rem;
+    z-index: 0;
+  }
+`;
+
+const TextBold = styled(Text)`
```
Comment (bulby97) [inline packages/core/admin/admin/src/components/UpgradePlanModal /index.js:52]: we don't need this compo we have `<Text bold />`

File: packages/core/admin/admin/src/components/UpgradePlanModal /index.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import { Text, H1 } from '@strapi/parts/Text';
+import { Stack } from '@strapi/parts/Stack';
+import ExternalLink from '@strapi/icons/ExternalLink';
+import CloseAlertIcon from '@strapi/icons/CloseAlertIcon';
+import AirBalloon from "../../assets/images/hot-air-balloon.png";
+import BigArrow from "../../assets/images/upgrade-details.png";
+import setOpacity from '../../utils/setOpacity';
+import useLockScroll from '../../utils/useLockScroll';
+
+const UpragdeWrapper = styled.div`
```
Comment (bulby97) [inline packages/core/admin/admin/src/components/UpgradePlanModal /index.js:20]: 'Upgrade' :)

File: packages/core/admin/admin/src/utils/useLockScroll.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
```
Comment (bulby97) [inline packages/core/admin/admin/src/utils/useLockScroll.js:1]: We should be it in the helper-plugin

File: packages/core/admin/admin/src/utils/setOpacity.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+const setOpacity = (hex, alpha) =>
```
Comment (bulby97) [inline packages/core/admin/admin/src/utils/setOpacity.js:1]: We should put it in helper plugin and rename it (something like setHexOpacity maybe)

**Review by bulby97 — CHANGES_REQUESTED** [review]
Good job :)

File: packages/core/helper-plugin/lib/src/utils/setHexOpacity.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const setHexOpacity = (hex, alpha) =>
```
Comment (soupette) [inline packages/core/helper-plugin/lib/src/utils/setHexOpacity.js:1]: Can you add a small story for this so we have some kind of documentation?

You can put it in the utils section.

File: packages/core/helper-plugin/lib/src/utils/useLockScroll.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+const useLockScroll = lockScroll => {
```
Comment (soupette) [inline packages/core/helper-plugin/lib/src/utils/useLockScroll.js:3]: This should be in the hooks folder. Can you also add a story?

**Review by soupette — CHANGES_REQUESTED** [review]

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**